CHANGELOG.md

```diff
@@ -5,6 +5,13 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/).
 
 ## [Unreleased]
 
+## [2022.3.0]
```

Comment:
Please note that putting this here is _not_ a release, and in fact can, and _has_ caused confusion of what is actually released.

There is an "Unreleased" section for a reason.

In this particular instance, I'm not interested in changing it because I _am_ pushing through a release immanently.

A release requires not only adding this line, but also updating the links at the bottom of the file and changing the VERSION file of the package itself. And most importantly, tagging the version so CI runs and uploads to PyPI